README.md

```diff
@@ -1,70 +1,68 @@
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
-
-### Analyzing the Bundle Size
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
-
-### Making a Progressive Web App
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
-
-### Advanced Configuration
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+# Songbox
+
+Songbox is a web service that allows users to listen to their music stored in Dropbox without having to install any app, directly from a web interface.
+
+## Features:
+
+1. Out of the box solution for mobile devices.
+2. Dropbox stream cross-platform player.
+3. Play music, mp3, audiobooks from Dropbox.
+4. Supported audio formats: mp3, wav, ogg.
+5. No need to install Dropbox app.
+6. Access to Dropbox audio content anywhere
+7. All your music, songs, audio, audio books, audio reports in one place.
+
+**Other:**
+
+1. Using HTML5 web browser.
+2. No need to install any additional software.
+3. No stored files in system memory.
+4. Free to use.
+
+## Webapp Installation
+```shell script
+$ git clone https://github.com/tepachelabs/songbox.webapp.git
+$ cd songbox.webapp
+$ npm install
+$ npm start
+
+# Open in your browser the url http://localhost:3000
+```
+## Backend URL
+Under the **src** folder there's a file called **config.js**. In this file you can change the **apiGatewayUrl** to fill your needs.
+whether you want to change the url for development or production (this is just the url for the [backend](https://github.com/tepachelabs/songbox.api))
+```
+const dev = {
+  apiGatewayUrl: 'http://localhost:4000',
+  ...
+};
+
+const prod = {
+  apiGatewayUrl: 'https://songbox.io',
+  ...
+};
+
+const stage = {
+  apiGatewayUrl: 'https://stage.songbox.io',
+  ...
+};
+```
+
+## Running Storybook
+Once you have installed all modules you can run the storybook. We added storybook, so we can see and test components in isolation.
```

Comment:
This sentence sounds repetitive. I think it could be just: `Once you have installed all modules, you can run Storybook to see and test components in isolation.`